restapi/header/header.avsc

```diff
@@ -4,8 +4,9 @@
   "name": "Header",
   "doc": "Metadata for Dataset. It is a summary explaining what kind of information are stored in the Dataset.",
   "fields": [
-    {"name": "descriptiveStatistic", "type": "DescriptiveStatistic" },
-    {"name": "unit", "type": "org.radarcns.avro.restapi.sensor.Unit"},
-    {"name": "effectiveTimeFrame", "type": "EffectiveTimeFrame" }
+    {"name": "descriptiveStatistic", "type": "DescriptiveStatistic", "doc": "Statical value expressed by samples" },
```

Comment:
The idea was that since the user needs to provide `userId`, `sourceId`, and `sensor` these 3 informations were redundant. While `descriptiveStatistic` is need to understand if the `value` of the returned sample is a `Quartile` or not.